### DISCUSSION TITLE
modal eror

### DISCUSSION THREAD

**Discussion by ardhrasyid** [filamentphp/filament#14915] [Help]
### Package

Actions

### Package Version

v3.0.0

### How can we help you?

https://github.com/user-attachments/assets/3370bcca-f094-4ae6-ac13-af0f5bc4c478

When I click delete, the delete modal does not appear correctly. When I create a new project, the same error still persists. And this issue seems to be affecting all modals.

**Comment by leandrocfe** [comment]
`php artisan about --only=Filament`

What is the output?

**Reply by leandrocfe** [reply]
Could you also share the code you are using?
